src/main/java/org/swisspush/redisques/RedisQues.java

```diff
@@ -166,7 +156,12 @@ public void start() {
 
         address = modConfig.getAddress();
         configurationUpdatedAddress = modConfig.getConfigurationUpdatedAddress();
-        redisPrefix = modConfig.getRedisPrefix();
+        redisPrefix           = modConfig.getRedisPrefix(); // default: "redisques:"
+        queuesKey             = redisPrefix + "queues"        ;
```

Comment:
Why this weird formatting for some variables?
Is this an editor feature?

Reply:
Yes, this is an editor feature - editor's name is "Oli" :-)
Column-aligned code when 'quasi-identical' stuff happens helps human eyes to see a pattern "oh - here happens x-times the same thing. Or nearly the same thing".
Differences in such patterns can quickly be identified.

Example:
```
        someObject.setSomeFancyProperty(sameObject.getSomeFunnyProperty());
        someObject.setSameOtherNotSoFancyProperty(someObject.getaFunnyProperty());
        someObject.setSomeFunnyProperty(sameObject.getaFunnyProperty());
        someObject.setaFunnyProperty(sameObject.getSomeFunnyProperty());
```
Do you see quickly that all four getter/setter statementa are (probably) wrong?
Now column aligned:
```
        someObject.setSomeFancyProperty          (sameObject.getSomeFunnyProperty());
        someObject.setSameOtherNotSoFancyProperty(someObject.getaFunnyProperty   ());
        someObject.setSomeFunnyProperty          (sameObject.getaFunnyProperty   ());
        someObject.setaFunnyProperty             (sameObject.getSomeFunnyProperty());
```
Now you see an additional typo

That's all: Repeated (and typically simple/stupid) lines of code are better human readable when identical sematic things are aligned.